### ISSUE TITLE
Old Gatsby v2 Image fixed/fluid fragments causes typegen error

### ISSUE THREAD

**Issue by jurreantonisse** [sanity-io/gatsby-source-sanity#114]
I Use gatsby-plugin-typegen to generate Sanity types from GraphQL. This worked great until v7.0.1. The codegen chokes on old Gatsby v2 Image fixed/fluid fragments in https://github.com/sanity-io/gatsby-source-sanity/blob/main/fragments/imageFragments.js.

The solution would be to remove these Gatsby v2 Image fixed/fluid fragments so we can use the newer versions of 
gatsby-source-sanity in production. I am happy to make a pull request if that makes things easier.

**Comment by TorbjornHoltmon** [comment]
I am having this issue too.
Typegen is spitting out `GraphQLDocumentError: Unknown type "SanityImageFluid". Did you mean "SanityImage", "SanityImageAsset", "SanityImageCrop", "SanityImageObject", or "SanityImagePalette"`

**Comment by jephjohnson** [comment]
Same issues - `Typegen is spitting out GraphQLDocumentError: Unknown type "SanityImageFluid". Did you mean "SanityImage", "SanityImageAsset", "SanityImageCrop", "SanityImageObject", or "SanityImagePalette"`

**Comment by waspeer** [comment]
I'm having the same issue:

```
GraphQLDocumentError: Unknown type "SanityImageFixed". Did you mean "SanityImageFit", "SanityImageAsset", "SanityImage", "SanityImageCrop", or "SanityImageCropEdge"?).
```

**Comment by ivoiv** [comment]
Getting the same error message on a fresh project.

packages 
```
"gatsby-plugin-typegen": "2.2.4"
"gatsby": "3.2.1"
"gatsby-source-sanity": "7.0.4"
```

gatsby-config.js
```
module.exports = {
  plugins: [
    `gatsby-plugin-typegen`,
    {
      resolve: "gatsby-source-sanity",
      options: {
        ...clientConfig.sanity
      },
    },
  ]
}
```

**Comment by SMccarrick** [comment]
I'm seeing this also

**Comment by philschonholzer** [comment]
Does someone have a workaround for this? Got the same problem.

**Comment by b0123498765** [comment]
Also seeing this.. hoping for a fix soon!

**Comment by jurreantonisse** [comment]
> Does someone have a workaround for this? Got the same problem.

Therese a pull request to fix this issue. However it's already there for over a month. I think we are all waiting for this to be merged https://github.com/sanity-io/gatsby-source-sanity/pull/120

**Comment by b0123498765** [comment]
Hosting a forked version with the PR merged for now.... https://github.com/dotdevv/gatsby-source-sanity

**Comment by rexxars** [comment]
Fixed in [v7.0.5](https://github.com/sanity-io/gatsby-source-sanity/releases/tag/v7.0.5)
